### PR TITLE
added nightly channel

### PR DESCRIPTION
**Pull request by mihaiplesa**
## Submitter Checklist:

- [ ] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [ ] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Windows
  - [ ] macOS
  - [ ] Linux
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on
  - [ ] Windows
  - [ ] macOS
  - [ ] Linux
- [ ] Verified that all lint errors/warnings are resolved (`npm run lint`)
- [x] Ran `git rebase master` (if needed).
- [x] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [ ] Add appropriate QA labels (QA/Yes or QA/No) to include the closed issue in milestone

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

### CODE REVIEW

**Comment by simonhong** [comment]
Wow, we will release nightly also? great :)

File: script/lib/omaha.py [hunk 1]
```diff
@@ -72,12 +72,14 @@ def get_base64_authorization(omahaid, omahapw):
 
 # To-Do: Add functions to create apps
 def get_appguid(channel, platform):
-    if channel in ['dev'] or platform in ['darwin'] and channel in ['beta', 'dev', 'release']:
+    if channel in ['dev'] or platform in ['darwin'] and channel in ['dev', 'beta', 'release']:
```
Comment (mihaiplesa) [inline script/lib/omaha.py]: should `nightly` be here as well @mbacchi?

File: script/lib/util.py [hunk 2]
```diff
@@ -254,6 +254,8 @@ def get_platform():
 def omaha_channel(platform, arch, preview):
     if platform == 'darwin':
         if preview:
+            if release_channel() in ['nightly']:
+                return 'test-nite'
```
Comment (mihaiplesa) [inline script/lib/util.py]: I still find `nite` to be more confusing than an abbreviation of `nightly`, perhaps have it as `ntly` or `nigh` (as we do `be` for `beta`)

Comment (mbacchi) [reply script/lib/util.py]: I think `nite` is more natural than `ntly` or `nigh`.

**Pull request merged**